sdks/python/src/honcho/async_client/session.py

```
@@ -0,0 +1,513 @@
+
+    async def add_peers(
+        self,
+        peers: str
+        | AsyncPeer
+        | tuple[str, SessionPeerConfig]
+        | tuple[AsyncPeer, SessionPeerConfig]
+        | list[AsyncPeer | str]
+        | list[tuple[AsyncPeer | str, SessionPeerConfig]]
+        | list[AsyncPeer | str | tuple[AsyncPeer | str, SessionPeerConfig]] = Field(
+            ..., description="Peers to add to the session"
+        ),
```

Ruff (0.11.9)

Do not perform function call `Field` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)